src/lt/objs/find.cljs

```diff
@@ -216,7 +216,7 @@
 (object/add-behavior! line-input ::exec-active!)
 
 (cmd/command {:command :goto-line
```

Comment:
Ive added 3 new PR containing the changes, one per each file to make sure everything is included... waiting feedback!